package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "chomtree",
```

Comment:
```suggestion
  "name": "chomTree",
```

Reply:
Follow up — do changes to package.json require package-lock.json to be regenerated?

Reply:
`package-lock.json` is regenerated whenever you run `npm install` or `npm update`.

Reply:
Also, only lowercase letters are allowed.

Reply:
:+1: